sql-queries/helpers/products.js

```diff
@@ -0,0 +1,17 @@
+import { PRODUCTS } from '../../constants'
+import squel from 'squel'
+
+export function getProductsIdsQuery(ids) {
+    let whereClause = ''
+    ids.forEach((item) => {
+        whereClause += `${PRODUCTS.COLUMNS.ID} = '${item}' OR `
+    } )
+    // SEEMS TO BE A VERY VERY BAD SOLUTION
```

Comment:
I wanna push my eyes with my fingers.
It's like: 
-I'll kill you. I know it's very bad, but I'll do it.